packages/lesswrong/components/common/LoadMore.tsx

```diff
@@ -61,7 +61,7 @@ const LoadMore = ({ loadMore, count, totalCount, className=null, disabled=false,
       href="#"
       onClick={handleClickLoadMore}
     >
-      Load More {totalCount && <span> ({count}/{totalCount})</span>}
+      Load More {totalCount ? <span> ({count}/{totalCount})</span> : ""}
```

Comment:
If `totalCount` is `0`, then this would render as "Load More 0" (because 0 is falsy, x&&y is x when x is falsy, and 0 is a special case that goes in the DOM even when it's falsy).

Reply:
ah.